src/constants.js

```diff
@@ -7,3 +7,12 @@
  * knowing their IDs.
  */
 export const NEWEST_MESSAGE_ANCHOR = Number.MAX_SAFE_INTEGER;
+
+/**
+ * Pointer to the oldest message in a narrow.
+ *
+ * Usually, an anchor is a message ID. This constant can be used when we
+ * want to retrieve the oldest messages for a narrow, without actually
+ * knowing their IDs.
+ */
+export const OLDEST_UNREAD_ANCHOR = 0;
```

Comment:
We do already use 'first unread' both in this project and in `zulip/zulip`.
I don't have any preference for this or your pick in a world where we haven't already picked one of the two. This is not that timeline :)
Let's just use `FIRST_UNREAD_ANCHOR` so we don't introduce inconsistent terminology.